views/default/elements/buttons.css.php

```diff
@@ -49,6 +49,7 @@
 	background: #0054a7 url(<?= elgg_get_simplecache_url("button_graduation.png"); ?>) repeat-x left 10px;
 }
 
+.elgg-button-submit:disabled,
```

Comment:
I think that's the jQuery pseudoselector. It should be `.elgg-button-submit[disabled]` in css